docs/assets/fragments/connectivity.txt

```diff
@@ -6,7 +6,7 @@
     user:
     
     ``` {.bash data-prompt="$" }
-    $ kubectl get secrets cluster1-secrets -o yaml -o jsonpath='{.data.root}' | base64 --decode | tr '\n' ' ' && echo " "
+    $ kubectl get secret cluster1-secrets -o jsonpath='{.data.root}' | base64 --decode && echo
```

Comment:
```suggestion
    $ kubectl get secret cluster1-secrets --template='{{"{{"}}.data.root | base64decode{{"}}"}}{{"{{"}}"\n"{{"}}"}}'
```

As far as I can recall, the `tr ...` part was needed for some exotic shells (zhs, etc.) to avoid extra newline.
Let's do it differently:  `kubectl get secrets cluster1-secrets --template='{{.data.root | base64decode}}{{"\n"}}'` uses some go templates magic within kubectl and it's short enough

Reply:
Thanks for this @fiowro , I tested on my mac and woks well.